luabsp.lua

```diff
@@ -451,13 +451,18 @@ do
             function(fl, lump_data)
                 lump_data.data = {}
                 lump_data.size = lump_data.filelen / 16
+                local origin = Vector(fl:ReadLong(), fl:ReadLong(), fl:ReadLong())
+                local size = fl:ReadLong()
+
+                if size < 1 then size = 6 end -- default size should be 32x32

```

Comment:
This block should be inside the for loop

Reply:
implemented this suggestion, since each subsequent cubemap after the first cubemap (index 0) would be the same as the first cubemap. this appears to be solved now